vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -73,9 +73,14 @@ def parse_node(node)
       new_result = parse_base_item(node)
 
       new_result.merge!(
-        :identity_infra   => node.spec.externalID,
-        :identity_machine => node.status.nodeInfo.machineID,
-        :identity_system  => node.status.nodeInfo.systemUUID
+        :identity_infra            => node.spec.externalID,
+        :identity_machine          => node.status.nodeInfo.machineID,
+        :identity_system           => node.status.nodeInfo.systemUUID,
+        :container_runtime_version => node.status.nodeInfo.containerRuntimeVersion,
+        :kernel_version            => node.status.nodeInfo.kernelVersion,
+        :kube_proxy_version        => node.status.nodeInfo.KubeProxyVersion,
+        :kubelet_version           => node.status.nodeInfo.kubeletVersion,
+        :os_image                  => node.status.nodeInfo.osImage
```

Comment:
Can we also avoid having kubernetes specific columns?  Do these specifically only even apply to kubernetes?  Something like a proxy seems like it could be common, so the kube prefix seems unecessary.